inspector/src/components/actionTabs/tabs/propertyGrids/materials/texturePropertyGridComponent.tsx

```diff
@@ -238,13 +312,28 @@ export class TexturePropertyGridComponent extends React.Component<ITextureProper
                             });
                         }} />
                     }
+                    <TextLineComponent label="Format" value={oformat?.label ?? "unknown"} />
+                    {
+                        !oformat?.hideType && !oformat?.compressed &&
+                        <TextLineComponent label="Type" value={otype?.label ?? "unknown"} />
+                    }
+                    {
+                        !!oformat?.normalizable && !oformat?.compressed && !!otype?.normalizable &&
```

Comment:
It gets pretty hard to read with so many signs :-)

Reply:
It would be even worst if I remove the ? and add some null/undefined checks :)